mods/simple_service.js

```diff
@@ -55,6 +55,9 @@ SimpleService.prototype.processSpec = function(spec) {
                 }
 
                 function backendRequest() {
+                    if (req.params.domain === 'en.wikipedia.test.local') {
```

Comment:
Not sure this is a good idea. We could circumvent this easily by fixing the domain part in the graphoid module request uri.